src/components/ToolMenu/FeatureInfo/index.tsx

```diff
@@ -112,10 +115,14 @@ export const FeatureInfo: React.FC<FeatureInfoProps> = ({
 
       const mapLayer = map.getAllLayers().find(l => {
         if (l instanceof OlLayerImage || l instanceof OlLayerTile) {
-          const unqualifiedMapLayerName = getUnqualifiedLayerName(l.getSource().getParams().LAYERS);
-          const unqualifiedLayerName = getUnqualifiedLayerName(layerName);
-
-          return unqualifiedLayerName === unqualifiedMapLayerName;
+          const source = l.getSource();
+          if (source instanceof OlSourceWMTS) {
+            return false;
+          } else {
+            const unqualifiedMapLayerName = getUnqualifiedLayerName(l.getSource().getParams().LAYERS);
+            const unqualifiedLayerName = getUnqualifiedLayerName(layerName);
+            return unqualifiedLayerName === unqualifiedMapLayerName;
+          }
```

Comment:
We might want to reuse the [isWmsLayer](https://github.com/terrestris/react-geo/blob/main/src/Util/typeUtils.ts#L10) guard from react-geo instead.